bn_testing/dags.py

```diff
@@ -20,6 +20,7 @@ class GroupedGaussianBN(object):
 
     Note:
         We assume that the order of the nodes is the topological ordering of the resulting graph
+        Comment az: There is more than one topological sorting !!!
```

Comment:
There is more than one topological sorting ?!

Reply:
You are right. Will reformulate.